contentcuration/contentcuration/models.py

```diff
@@ -1700,18 +1700,19 @@ def get_details(self, channel_id=None):
             "resource_count": node.get("resource_count", 0),
             "resource_size": node.get("resource_size", 0),
             "includes": for_educators,
-            "kind_count": node.get("kind_count", []),
-            "languages": node.get("languages", ""),
-            "accessible_languages": node.get("accessible_languages", ""),
-            "licenses": node.get("licenses", ""),
-            "tags": node.get("tags_list", []),
-            "copyright_holders": node["copyright_holders"],
-            "authors": node["authors"],
-            "aggregators": node["aggregators"],
-            "providers": node["providers"],
-            "sample_pathway": pathway,
+            "kind_count": node.get("kind_count") or [],
+            "languages": node.get("languages") or [],
+            "accessible_languages": node.get("accessible_languages") or [],
+            "licenses": node.get("licenses") or [],
+            "tags": node.get("tags_list") or [],
             "original_channels": original_channels,
+            "sample_pathway": pathway,
             "sample_nodes": sample_nodes,
+            # source model fields for the below default to an empty string, but can also be null
+            "authors": list(filter(bool, node["authors"])),
```

Comment:
Ah yeah - I fixed this on the frontend, but better to just not return this data at all.